frontend/src/queries/nodes/DataVisualization/Components/Variables/Variables.tsx

```diff
@@ -93,14 +102,23 @@ const VariableInput = ({
     onRemove,
     variableSettingsOnClick,
 }: VariableInputProps): JSX.Element => {
-    const [localInputValue, setLocalInputValue] = useState(() => {
+    const [calendarTime, setCalendarTime] = useState<boolean>(false)
+    const [localInputValue, setLocalInputValue] = useState<string>(() => {
         const val = variable.value ?? variable.default_value
 
         if (variable.type === 'Number' && !val) {
-            return 0
+            return '0'
```

Comment:
Was it just stringified before?

Reply:
it was number type all the way through before this is just making the typing consistent. When it's passed to the query it should still be a number!